pkgs/development/python-modules/simpy/default.nix

```
@@ -9,10 +10,17 @@ buildPythonPackage rec {
     sha256 = "dd8c16ca3cff1574c99fe9f5ea4019c631c327f2bdc842e8b1a5c55f5e3e9d27";
   };
 
+  nativeBuildInputs = [ setuptools_scm setuptools ];
```

provide as part of buildPythonPackage
```suggestion
  nativeBuildInputs = [ setuptools_scm ];
```